### PR TITLE
Add common/error page component

### PR DESCRIPTION
**Pull request by MieczyslawFrontDev**
Hi !

I created an ErrorPage component and put it in the common folder. It will be used and handle a problems that could ocurred with getting data from api wherever any will be fetched. 
The component displays an error message and information about what might be the reason.
Also includes a button that allows you to return to the application's home page.

### CODE REVIEW

File: src/common/ErrorPage/styled.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+`;
+
+export const Button = styled.button`
+  padding: 16px 24px;
+  font-size: 14px;
+  font-weight: 700;
+  color: ${({ theme }) => theme.colors.white};
+  background-color: ${({ theme }) => theme.colors.blue};
+  border-radius: 5px;
+
```
Comment (paulgrym) [inline src/common/ErrorPage/styled.js:57]: I think Button would look more similiar to the design with border set to none :D
Maybe we can also add cursor set to pointer and some hover state?

File: src/common/ErrorPage/styled.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
```
Comment (paulgrym) [inline src/common/ErrorPage/styled.js:58]: Media for small screens looks good! 😀 Maybe a little adjustements for medium screens - there is no left and right padding there and text is quite big? :D

**Review by paulgrym — COMMENTED** [review]
Well done :D

File: src/common/ErrorPage/styled.js (hunk 1 above)
Comment (MieczyslawFrontDev) [reply src/common/ErrorPage/styled.js:57]: Ok, I will do it ;)

File: src/common/ErrorPage/styled.js (hunk 2 above)
Comment (MieczyslawFrontDev) [reply src/common/ErrorPage/styled.js:58]: Thanks for suggest ;)

**Comment by MieczyslawFrontDev** [comment]
I've corrected the view as suggested.

**Pull request merged**